Use FreeBSD's Expat (a.k.a. bsdxml) instead of the bundled version.

	-mi

--- configure	2017-08-24 09:59:41.657305804 -0400
+++ configure	2018-02-05 14:36:36.444830000 -0500
@@ -5443,8 +5443,8 @@
                 ;;
             *)
-                if test -f "$with_expat/include/expat.h"; then
+                if test -f "$with_expat/include/bsdxml.h"; then
                     ac_cv_c_expat=`(cd $with_expat; pwd)`
                 else
-                     as_fn_error $? "${with_expat} directory doesn't contain expat.h" "$LINENO" 5
+                     as_fn_error $? "${with_expat} directory doesn't contain bsdxml.h" "$LINENO" 5
                 fi
         esac
@@ -5511,2 +5511,2 @@
 
-    vars="-lexpat"
+    vars="-lbsdxml"
--- generic/dom.h	2017-08-23 10:19:33.726158496 -0400
+++ generic/dom.h	2018-02-05 14:34:53.308599000 -0500
@@ -39,5 +39,5 @@
 #include <tcl.h>
 #include <ctype.h>
-#include <expat.h>
+#include <bsdxml.h>
 #include <utf8conv.h>
 
--- generic/tdom.h	2017-08-23 10:19:33.726158496 -0400
+++ generic/tdom.h	2018-02-05 14:34:53.308798000 -0500
@@ -1,5 +1,5 @@
 
 #include "tcl.h"
-#include <expat.h>
+#include <bsdxml.h>
 
 
--- generic/tclexpat.h	2017-08-23 10:19:33.725158500 -0400
+++ generic/tclexpat.h	2018-02-05 14:34:53.309022000 -0500
@@ -1,4 +1,4 @@
 #include <tcl.h>
-#include <expat.h>
+#include <bsdxml.h>
 
 struct TclGenExpatInfo;
